ui/src/components/common/SubState.js

```diff
@@ -0,0 +1,103 @@
+import { set as _set, get as _get } from 'lodash-es';
+import { VALIDATION_WARN, VALIDATION_OK } from './Validation';
+
+export class SubState {
```

Comment:
do we really need this class if each components manage its own state?

Reply:
removed file